spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/config/AdminServerProperties.java

```diff
@@ -127,6 +129,56 @@ public String path(String path) {
 
 	}
 
+	@lombok.Data
+	public static class InstanceAuthProperties {
+
+		/**
+		 * Whether or not to use configuration properties as a source for instance
+		 * credentials <br/>
+		 * Default: false
+		 */
+		private boolean enabled = false;
+
+		/**
+		 * Default username used for authentication to each instance. Individual values
+		 * for specific instances can be overriden using
+		 * `spring.boot.admin.instance-auth.service-map.*.user-name`. <br/>
+		 * Default: client
+		 */
+		private String defaultUserName = "client";
```

Comment:
Imho the default value must be `null`. We should not provide default usernames/passwords

Reply:
Set to null